### DISCUSSION TITLE
Is code annotations in call out blocks supported

### DISCUSSION THREAD

**Discussion by cudawarped** [quarto-dev/quarto-cli#5484] [Q&A] [answered]
I can use [code annotation](https://quarto.org/docs/authoring/code-annotation.html#annotation-syntax) inside a single [callouts](https://quarto.org/docs/authoring/callouts.html#markdown-syntax) block without any issues, however if I try to use it in a second call outblock it fails.  e.g. the "more addition" annotation is not shown in

````
---
format: html
---

::: {.callout-tip}
            
Python annotation 1

```{.python}
   1 + 1 # <1>
```
1. addition

:::
            
::: {.callout-tip}
            
Python annotation 2

```{.python}
   1 + 1 # <1>
```
1. more addition

:::
````

**Comment by cscheid** [comment, marked as answer]
I can repro this, it's a bug. I'll create an issue.

**Comment by mcanouil** [comment]
@cudawarped For possible future post, please don't add `>` to the code you share, it makes it impossible to use without having to manually remove the `>` manually.
Also, please share a complete Quarto document, especially because the format matters.

Finally, thanks for your report!

See an example:

````qmd
---
format: html
---

::: {.callout-tip}
            
Python annotation 1

```{.python}
   1 + 1 # <1>
```
1. addition

:::
            
::: {.callout-tip}
            
Python annotation 2

```{.python}
   1 + 1 # <1>
```
1. more addition

:::
````

**Reply by cudawarped** [reply]
> @cudawarped For possible future post, please don't add `>` to the code you share, it makes it impossible to use without having to manually remove the `>` manually. Also, please share a complete Quarto document, especially because the format matters.

No worries, how to I enclose the raw markdown so that it can be displayed correctly?

**Reply by cscheid** [reply]
The number of ticks on the outside needs to always be bigger than the number of ticks on the inside.

**Reply by mcanouil** [reply]
There is an example in the bug report guide, see <https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us>.

Below, I am even using five ` ````` `:

`````
````
This is a four backticks block.

```
This is a code block
```
````
`````

**Reply by cudawarped** [reply]
Excellent, updated original post for good measure.

**Reply by mcanouil** [reply]
Thank you!

**Comment by mcanouil** [comment]
Closing this.
For follow-up please check the related issue #5485.
